zaza/openstack/charm_tests/manila_ganesha/tests.py

```diff
@@ -56,3 +58,65 @@ def _restart_share_instance(self):
                     unit.entity_id,
                     "systemctl stop manila-share nfs-ganesha")
         return True
+
+    def _run_nrpe_check_command(self, commands):
+        try:
+            zaza.model.get_application("nrpe")
+        except KeyError:
+            self.skipTest("Skipped NRPE checks since nrpe is not deployed.")
+
+        units = zaza.model.get_units("manila-ganesha-az1")
+
+        for attempt in tenacity.Retrying(
+            wait=tenacity.wait_fixed(20),
+            stop=tenacity.stop_after_attempt(2),
+            reraise=True,
+        ):
+            with attempt:
+                ret = generic_utils.check_commands_on_units(commands, units)
+                self.assertIsNone(ret, msg=ret)
+
+    def test_903_nrpe_custom_plugin_checks(self):
+        """Confirm that the NRPE custom plugin files are created."""
+        plugins = [
+            "check_nfs_conn",
+            "check_nfs_exports",
+            "check_nfs_services",
+        ]
+
+        commands = [
+            f"ls /usr/local/lib/nagios/plugins/{plugin}"
```

Comment:
From the py35 test run:

```
zaza/openstack/charm_tests/manila_ganesha/tests.py:88:56: E999 SyntaxError: invalid syntax
```

Py3.5.sadly doesn't support f-strings.  You'll need to use `.format(...)` here instead.

Reply:
Done, thanks for the hints!